zng/resolver/mapper.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 type Mapper struct {
-	Slice
+	types     []zng.Type
 	mu        sync.RWMutex
 	outputCtx *zson.Context
```

Comment:
Nit: `mu` protects `types` but not `outputCtx`, so group and order accordingly.
```suggestion
	outputCtx *zson.Context

	mu        sync.RWMutex
	types     []zng.Type
```